modules/registry-common-terms.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * registry/index.adoc
+
+:_content-type: REFERENCE
+[id="openshift-registry-common-terms_{context}"]
+= Common terms
+
+This glossary defines common terms that are used in the architecture content. These terms help you understand {product-title} architecture effectively.
+
+container::
```

Comment:
Better to use uppercase letter for these iterms

Reply:
As per IBM style guide suggestion[1] we should keep them lower case letters 

[1] `Do not capitalize a term in the glossary unless it is a proper noun.`